test-in-docker/Dockerfile

```diff
@@ -1,24 +1,37 @@
 FROM debian:10 AS builder
 
-RUN apt update && apt install -y unzip libpcap-dev build-essential git cmake && rm -rf /var/lib/dpkg/lists/*
+RUN apt-get update && apt-get install -y \
+   unzip \
+   libpcap-dev \
+   build-essential \
+   git \
+   cmake \
+   wget \
+   && rm -rf /var/lib/dpkg/lists/*
 
 WORKDIR /src
 
 RUN git clone https://git.savannah.nongnu.org/git/lwip.git
 
-ADD http://elm-chan.org/fsw/ff/arc/ff14.zip .
-RUN unzip -q ff14.zip -d ff
+RUN wget http://elm-chan.org/fsw/ff/arc/ff14.zip && unzip -q ff14.zip -d ff && rm ff14.zip
```

Comment:
This optimization may make the Dockerfile more idiomatic but is unnecessary here: The completed image will not contain any layers from the builder.